plutus-tx/src/PlutusTx/AssocMap.hs

```diff
@@ -149,6 +152,10 @@ instance (Pretty k, Pretty v) => Pretty (Map k v) where
 fromList :: [(k, v)] -> Map k v
 fromList = Map
 
+{-# INLINEABLE fromListSafe #-}
+fromListSafe :: Eq k => [(k, v)] -> Map k v
```

Comment:
unused?

Reply:
Yep, but I believe it's worth having it if just to emphasize that the default version isn't safe.